client/signup/style.scss

```diff
@@ -784,8 +788,24 @@ body.is-section-signup.is-white-signup .layout:not( .dops ):not( .is-wccom-oauth
 		}
 	}
 
+	.signup__step.is-design,
+	.signup__step.is-design-setup-site {
+		.step-wrapper__header {
+			@include breakpoint-deprecated( '<660px' ) {
+				margin-left: 0;
+				margin-right: 0;
+			}
+		}
+	}
+
 	button.is-primary {
 		box-shadow: none;
 		border: 0;
 	}
+
+	.inline-help {
```

Comment:
Move up the inline-help button under the reskinned flow on mobile because we have sticky navigation buttons. (Not sure how to check we have sticky navigation buttons)